images/base/Dockerfile

```diff
@@ -116,9 +116,10 @@ RUN eval "$(gimme "${GO_VERSION}")" \
 
 
 # stage for building containerd
+# set by makefile to the desired containerd version
+ARG CONTAINERD_VERSION
 FROM go-build as build-containerd
 ARG TARGETARCH GO_VERSION
-ARG CONTAINERD_VERSION="v1.7.5"
```

Comment:
this is inconsistent with every other version in the file ..?

Reply:
i was mirroring `ARG GO_VERSION` in line 113 above.

Reply:
why don't keep the default version here?
Also, what is the benefit, this is more a personal commodity no? I don't imaging now we are going to release multiple images with different containerd versions

Reply:
@aojea this is to test newer versions of containerd in CI :)

Reply:
The go version had to be moved out for consistency with the rest of the repo, but the containerd version only exists here.

You can already do `EXTRA_BUILD_OPT` to pass additional args to the dockerfile, and we can already PR updates to the CONTAINERD_VERSION.

Reply:
Ack, was trying to make it more obvious/explicit with this PR. will just try to use `EXTRA_BUILD_OPT`